apps/wallet/src/ui/app/shared/page-main-layout/PageMainLayout.tsx

```diff
@@ -7,7 +7,6 @@ import cn from 'clsx';
 import { createContext, type ReactNode, useState } from 'react';
 import { useAppSelector, useActiveAccount } from '_hooks';
 import { AppType } from '../../redux/slices/app/appType';
-import { DappStatus } from '../dapp-status';
```

Comment:
shouldnt we keep cleaning? eg `apps/wallet/src/ui/app/shared/dapp-status` that whole folder isnt used anymore, and then that folder uses other parts of the code which could also be deleted like `createDappStatusSelector`, `ButtonConnectedTo`, ...

Reply:
fair